controllers/dopplersecret_controller_secrets.go

```diff
@@ -214,10 +192,7 @@ func (r *DopplerSecretReconciler) UpdateManagedSecret(ctx context.Context, secre
 	if procsVersErr != nil {
 		return fmt.Errorf("Failed to compute processors version: %w", procsVersErr)
 	}
-	nameTransformersVersion, transVersErr := GetNameTransformerVersion(dopplerSecret.Spec.NameTransformer)
-	if transVersErr != nil {
-		return fmt.Errorf("Failed to compute name transformers version: %w", transVersErr)
-	}
+	nameTransformersVersion := dopplerSecret.Spec.NameTransformer
```

Comment:
I think we can drop `nameTransformersVersion` now as well. In the past they were needed because if we changed the name transformer, we needed to drop the ETag from the request so we could force-fetch the secret data and we could re-do the transform. With the new API changes, a change to the name transformer will result in a new ETag so we don't need special logic 🎉

Reply:
Done!